hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAsyncReplicationAdminApi.java

```diff
@@ -369,18 +365,17 @@ public void testSetPeerNamespaces() throws Exception {
     String ns1 = "ns1";
     String ns2 = "ns2";
 
-    ReplicationPeerConfig rpc = new ReplicationPeerConfig();
-    rpc.setClusterKey(KEY_ONE);
+    ReplicationPeerConfig rpc = ReplicationPeerConfig.newBuilder().setClusterKey(KEY_ONE).build();
     admin.addReplicationPeer(ID_ONE, rpc).join();
-    rpc.setReplicateAllUserTables(false);
+    rpc = ReplicationPeerConfig.newBuilder(rpc).setReplicateAllUserTables(false).build();
```

Comment:
Rather than creating a new builder from `rpc`, we can reuse old one.
e.g
```
ReplicationPeerConfigBuilder rpcBuilder = ReplicationPeerConfig.newBuilder().setClusterKey(KEY_ONE);
admin.addReplicationPeer(ID_ONE, rpcBuilder.build()).join();
rpcBuilder.setReplicateAllUserTables(false);
admin.updateReplicationPeerConfig(ID_ONE, rpcBuilder.build()).join();
```

Applicable to other similar places.

Reply:
Thanks for the nice suggestion, fix it immediately.